### PR TITLE
t: Add coverage for _stop_video_encoder

### PR DESCRIPTION
**Pull request by perlpunk**
Issue: https://progress.opensuse.org/issues/105984

### CODE REVIEW

**Comment by perlpunk** [comment]
Reported the flaky `consoles/virtio_terminal.pm` in https://progress.opensuse.org/issues/105759

**Comment by perlpunk** [comment]
Created https://progress.opensuse.org/issues/107254 for consoles/VNC.pm (I checked if my change can cause this but apparently it doesn't)

File: t/23-baseclass.t [hunk 1]
```diff
@@ -308,6 +309,57 @@ subtest 'SSH utilities' => sub {
...
+    for (1 .. 3) {
+        my $pipe = IO::Pipe->new;
+        my $pid = fork;
+        if ($pid) { $pipe->writer }
+        elsif (defined $pid) {
+            $pipe->reader;
+            my @lines = <$pipe>;
+            exit;
+        }
+        else { die "Couldn't fork" }
```
Comment (Martchus) [inline t/23-baseclass.t:323]: Maybe this line should be marked as uncoverable.

File: t/23-baseclass.t [hunk 2]
```diff
@@ -308,6 +309,57 @@ subtest 'SSH utilities' => sub {
...
+sub _prepare_video_encoder ($baseclass) {
+    my @pipes;
+    for (1 .. 3) {
+        my $pipe = IO::Pipe->new;
+        my $pid = fork;
+        if ($pid) { $pipe->writer }
+        elsif (defined $pid) {
+            $pipe->reader;
+            my @lines = <$pipe>;
+            exit;
```
Comment (Martchus) [inline t/23-baseclass.t:321]: These lines will likely not be tracked so they should likely be marked as uncoverable.

Comment (perlpunk) [reply t/23-baseclass.t]: ok. we're not collecting coverage for `t/` yet, but I marked the lines and hope I got it right

**Review by okurz — APPROVED** [review]

**Review by mimi1vx — APPROVED** [review]

**Pull request merged**